engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/sink/HardcodedSchemaTableSink.scala

```diff
@@ -13,7 +13,7 @@ import pl.touk.nussknacker.engine.flink.table.utils.TableUtils.buildTableDescrip
 import pl.touk.nussknacker.engine.flink.table.DataSourceConfig
 import pl.touk.nussknacker.engine.flink.table.utils.HardcodedSchema
 
-class TableSink(config: DataSourceConfig, value: LazyParameter[java.util.Map[String, Any]]) extends FlinkSink {
+class HardcodedSchemaTableSink(config: DataSourceConfig, value: LazyParameter[java.util.Map[String, Any]]) extends FlinkSink {
```

Comment:
We don't need this hardcoded components anymore - please remove them. Also, please move valuable things from there to the current implementation. E.g. we developed a nice documentation in the `HardcodedSchemaTableSink`, which is missing in the copied `SqlSink` version.

Reply:
Done :+1: